messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "Die Dorfbewohner entscheiden sich {0:@} zu hängenfffff das Seil dehnt sich und reist, und der folgende Sturz tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner haben es schwer mit dem Schmerz des Todes von {0:@} und lynchten wiederwillig {1!role:article} {1!role:bold}.",
+        "Konform mit dem Willen des Dorfes, erwiesen sich die Galgen als wirksam bei der Tötung von {0:@}, {1!role:article} {1!role:bold}.",
+        "In einem Anfall von Hysterie lynchen die Dorfbewohner {0:@}, und töten {1!role:article} {1!role:bold}.",
+        "Während sie auf ihre Hinrichtung warten, fällt ein Baum auf {0:@}, dies tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner fesseln {0:@}. Und werfen {0:@} in einen See. Es scheint so das {1!role:article} {1!role:bold} nicht gut schwimmen kann",
+        "Glaubt man ihrem Nachbarn und Freund das {0:f@} gefährlich zu sein scheint, so legt die Masse der Dorfbewohner den {1!role:bold} {0:@} um.",
+        "Unter viel Lärm lynchen die Pechgabel-lagerischen Dorfbewohner {0:@}, was sich als... {1!role:article} {1!role:bold} entpuppte.",
+        "Trotz Protesten schleppt der Mob sein Opfer an den hängenden Baum. {0:@} erliegt dem Willen der Horde und wird gehängt. Die Dorfbewohner haben {1!role:article} {1!role:bold} getötet .",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Sobald das Zucken aufhörte, wurde entdeckt, dass das Dorf {1!role:article} {1!role:bold} gelyncht hat.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Danach inspiziert das Dorf die Leiche und findet {1!role:article} {1!role:bold}.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Leider reist das Seil, und {1!role:article} {1!role:bold} schlägt auf den Boden auf und stirbt.",
+        "Bevor das Seil gezogen wird, {0:@}, {1!role:article} {1!role:bold}, zieht den Stift an einer Granate. Sie zögert, und explodiert und tötet {0:@}.",
+        "Bevor das Seil gezogen wird wirft {0:@} {1!role:article} {1!role:bold}, eine Granate auf Menge doch die Granate explodierte zu früh.",
+        "Die Dorfbewohner ziehen {0:@} an den Rand einer Klippe. Bevor sie protestieren können, {1!role:article} {1!role:bold} rutscht und fällt in den Abgrund.",
+        "Mit einem schaurigen Kriegsschrei ziehen die Dorfbewohner {0:@} an den Rand einer Klippe und werfen {0:@} hinab. Die Felswand ist nun mit den Rester von {0:@} {1!role:article}  {1!role:bold} bedeckt.",
+        "{0:@} wird auf ein Rudel hungriger Tiere geworfen. das {1!role:bold} leben hält nicht lange.",
+        " {0:@} into a volcano. They discover the melting point of {1!role:article} {1!role:bold} is less than that of lava.",
+        "Für die WISSENSCHAFT, die Dorfbewohner fesseln {0:@} an einen Elektrischen Stuhl und Schalten den Strom ein. Der Verbrannte Körper identifizierte sich dann als {1!role:article} {1!role:bold}.",
```

Comment:
not sure "For SCIENCE" can be translated idiomatically. Might remove this line

Reply:
for SCIENCE is german Für die WISSENSCHAFT.

Reply:
An idiom is some phrase that has a specific meaning that may not be readily apparent just from the words. When attempting to translate an idiom, keep in mind that the target language may not have anything of the sort. This particular phrase in English is a bit of a meme and is meant to indicate doing something without regards for consequences that you know will probably be bad. See https://tvtropes.org/pmwiki/pmwiki.php/Main/ForScience for more information.

Running things through Google only gives you half of the story, you need to understand the words in context as well. As I mentioned previously, this section doesn't need to get translated 1:1 -- there are cultural references in here that really won't make sense in German, and this is an opportunity to add in some German cultural references as well. If this phrase doesn't have a German equivalent _with the same underlying meaning_ (not just the same words, the same meaning), then it's better to remove it entirely.

Reply:
There is one page, that explains these things pretty well IMO. It's Urban Dictionary.

> An exclamation added after suggesting an idea that is obviously terrible, or a request with sexual undertones.
>
>1. You know beer pong, where you have a drink for each point? We should try beer-scrabble!... for science!
>
> 2. so you used to do a pole-dancing workout? you should send me a video of that.. for science!

https://www.urbandictionary.com/define.php?term=for%20science

Reply:
Danke Skizzerz ich werden in zukunft bei zeilen bei denen ich keine eindeutige übersetzungen bzw falsche informationen erhalte diese entfernen wenn es okay ist allerdings entspricht es dann nicht mehr der orginalen en.json weil dann vllt einige zeiilen fehlen. bzw wörter